src/app/containers/Figure/index.jsx

```diff
@@ -45,25 +50,53 @@ const FigureContainer = ({
     <Image alt={alt} src={src} width={width} srcset={srcset} />
   );
 
+  let Wrapper = NestedGridItemLarge;
+
+  if (height === width) {
+    Wrapper = NestedGridItemMedium;
+  }
+  if (height > width) {
+    Wrapper = NestedGridItemSmall;
+  }
+
   return (
-    <Figure>
-      <ImagePlaceholder ratio={ratio}>
-        {platform === 'amp' ? (
-          <AmpImg
-            alt={alt}
-            attribution={copyright || ''}
-            layout="responsive"
-            src={src}
-            height={height}
-            width={width}
-          />
-        ) : (
-          renderImage(imageToRender, lazyLoad)
-        )}
-        {renderCopyright(copyright)}
-      </ImagePlaceholder>
-      {renderCaption(captionBlock, type)}
-    </Figure>
+    <Wrapper
+      gridColumnStart={1}
+      gridSpan={{
+        group3: '10',
+        group4: '5',
+        group5: '10',
+      }}
+    >
```

Comment:
Here you should be using one of these parent items: `NestedGridParentLarge`/ `NestedGridParentMedium`/`NestedGridParentSmall`, depending on ratio. So, for example, defining `ParentWrapper`. Then the wrapper using `NestedGridItemLarge`/`NestedGridItemMedium`/`NestedGridItemSmall` can be called `ChildWrapper`. 

You can then use `ChildWrapper` twice - once to surround the Caption (which you're doing currently) and once to surround the ImagePlaceholder.

Reply:
so, in general, we should have 3 wrappers 
one ParentWrapper wrapping the entire Figure container
one ChildWrapper wrapping the ImagePlaceholder
and another wrapping the Caption? @sareh